File: practicaid/src/main/java/com/prueba/inyeccion/practicaid/Models/Domain/Cliente.java
package com.prueba.inyeccion.practicaid.Models.Domain;

import org.springframework.beans.factory.annotation.Value;
import org.springframework.stereotype.Component;

@Component
public class Cliente {

    @Value("${cliente.nombre}")
    private String nombre;
    @Value("${cliente.apellido}")
    private String apellido;

    public String getNombre() {
        return nombre;
    }

    public void setNombre(String nombre) {
        this.nombre = nombre;
    }

    public String getApellido() {
        return apellido;
    }

    public void setApellido(String apellido) {
        this.apellido = apellido;
    }

}

File: Angular/clientes-app/src/app/modelo/cliente.ts
export class Cliente {
  id: number;
  nombre: string;
  apellido: string;
  createAt: string;
  email: string;

  constructor(
    id: number,
    nombre: string,
    apellido: string,
    createAt: string,
    email: string
  ) {
    this.id=id;
    this.nombre=nombre;
    this.apellido=apellido;
    this.createAt=createAt;
    this.email=email;
  }

  
}
